src/plugins.json

```diff
@@ -493,6 +493,11 @@
     "description": "Add an id and anchor to all headings on a page. Ideal for permalinks."
   },
   {
+    "name": "Hooks",
+    "repository": "https://github.com/benwilhelm/metalsmith-hooks",
```

Comment:
This is missing an "icon" attribute. (If you're not sure what to put, check out the other plugins on metalsmith.io and copy one of the icons from those.)